README.md

```diff
@@ -178,10 +178,13 @@ The credentials files use YAML syntax, and the usage will vary depending on the
     #   username: admin
     #   password: password
 
-Custom report
--------------
+HTML report
+-----------
 
-By default a custom HTML report will be written to results/index.html. If you wish this to be located elsewhere, or have a different filename, you can specify the --webqareport command line option.
+If the [pytest-html](https://pypi.python.org/pypi/pytest-html) plugin is installed
+then the HTML reports will include additional information such as the failing URL,
+screenshot, and page source. For Sauce Labs, a link to the job and inline video
+will also be included.
```

Comment:
That makes sense. Perhaps a mention that a command line argument is required to generate the report with a link to the docs for pytest-html?